Documentation/ApiOverview/SiteHandling/Basics.rst

```diff
@@ -181,3 +191,37 @@ routeEnhancers
 While page routing works out of the box without any further settings, route
 enhancers allow configuring routing for TYPO3 extensions. Read more at
 :ref:`routing-advanced-routing-configuration`.
+
+
+.. _sitehandling-basics-settings:
+
+settings
+--------
+
+The settings section can be used to define custom site settings. These settings
+can be used in PHP code, TypoScript and Fluid templates. Read more at
+:ref:`sitehandling-settings`.
+
+..  note::
+    If folder contains a `settings.yaml` file, all `settings` in `config.yaml`
```

Comment:
Strange, you suggest to delete the settings.yaml so that the settings from config.yaml are considered again. And then you hint, but next time you use the editor, the settings.yaml is here again.

Better would be to remove the settings from config.yaml.